python/cudf/cudf/core/reshape.py

```diff
@@ -915,10 +916,10 @@ def pivot(data, index=None, columns=None, values=None):
 
     Parameters
     ----------
-    index : column name, optional
-        Column used to construct the index of the result.
     columns : column name, optional
         Column used to construct the columns of the result.
+    index : column name, optional
```

Comment:
Nit: If this is internal maybe this param ordering can be kept?

Reply:
This is actually a public api that can be accessed by `cudf.pivot`

Reply:
Ah okay I see that it matches `pd.pivot` as well